src/pages/errors/[errorCode].tsx

```diff
@@ -36,7 +36,7 @@ export default function ErrorDecoderPage({
         }}
         routeTree={sidebarLearn as RouteItem}
         section="unknown">
-        {parsedContent}
+        <div style={{whiteSpace: 'pre-line'}}>{parsedContent}</div>
```

Comment:
Can you use the tailwind class for this actually?

Reply:
Of course, sorry, I knew better! Changes submitted!